src/main/java/com/github/manolo8/darkbot/backpage/entities/galaxy/GatesList.java

```diff
@@ -0,0 +1,32 @@
+package com.github.manolo8.darkbot.backpage.entities.galaxy;
+
+public enum GatesList {
```

Comment:
don't name an "enum" a "list", it's confusing. Also, this isn't all the gates wich the name "Gates" may make you think. It should have a name more closely representing it, like SpinGate or something along the lines.